src/main/java/com/spring/rest/dao/impl/CarCharacteristicsDaoImpl.java

```diff
@@ -29,10 +30,10 @@
     @Value("${car.characteristics.get}")
     private String GET_CAR_CHARACTERISTICS_SQL;
     /**
-     * SQL query to get cars characteristics.
+     * SQL query to get list of cars characteristics.
      */
-    @Value("${cars.characteristics.get}")
-    private String GET_CARS_CHARACTERISTICS_SQL;
+    @Value("${list.cars.characteristics.get}")
```

Comment:
`car.characteristics.list.get` is better as for me.

Reply:
should I write constants in the same order as a property name?